static_src/components/base_component.jsx

```diff
@@ -0,0 +1,8 @@
+
+import React from 'react';
+
+export default class BaseComponent extends React.Component {
```

Comment:
No purpose right now, I should take it out. I think I was experimenting with it and found a better way to do something.